README.md

```diff
@@ -69,57 +67,60 @@ Below are the methods this SDK exposes. See [Iterable's API Docs](https://api.it
+| handleLinks                  | How to open links. If `undefined`, use browser-default behavior. `open-all-new-tab` opens all in new tab, `open-all-same-tab` opens all in same tab, `external-new-tab` opens only off-site links in new tab, otherwise same tab. Overrides the target attribute defined on link elements. | `'open-all-new-tab' \| 'open-all-same-tab' \| 'external-new-tab'` | `undefined` |
+| onOpenScreenReaderMessage    | What text do you want the screen reader to announce when opening in-app messages                                                                                                                                                                                                           | `string`                                                          | `undefined` |
+| onOpenNodeToTakeFocus        | What DOM element do you want to take keyboard focus when the in-app message opens. (Will open the first interact-able element if not specified). Any query selector is valid.                                                                                                              | `string`                                                          | `undefined` |
+| rightOffset                  | How much space (px or %) to create between the right of the screen and messages. Not applicable for center or full-screen messages                                                                                                                                                         | `string`                                                          | `undefined` |
+| topOffset                    | How much space (px or %) to create between the top of the screen and messages. Not applicable for center, bottom, or full-screen messages                                                                                                                                                  | `string`                                                          | `undefined` |
+| closeButton :rotating_light: | Properties to show a custom close button on each in-app message                                                                                                                                                                                                                            | `CloseButtonOptions` (see below)                                  | `undefined` |
 
-:rotating_light: Due to a limitation in Safari, web in-app messages displayed in Safari do not respect the value set for `handleLinks`. As a workaround, give your links a `target` (`target="_blank"` for new tab, `target="_top"` / `target="_parent"` for same tab).
+:rotating_light: Due to a limitation in Safari browsers, web in-app messages displayed in Safari will not be able to add an auto-generated close button and automatically fire `trackInAppClose` events with it for you.
```

Comment:
@pauljung14 Can safari create the button, but not fire the events? Or neither? Can we clarify this?

Reply:
Got it, will clarify